### PR TITLE
chore: update .gitignore

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: ui/.gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+/build/
```
Comment (g1eny0ung) [inline ui/.gitignore:7]: I think this is due to the old production build. Move it to the below `# production` section?

```
# production
/build/
/app/build
```

**Review by g1eny0ung — APPROVED** [review]
Rest LGTM.

File: ui/.gitignore (hunk 1 above)
Comment (cwen0) [reply ui/.gitignore]: Fixed

File: .gitignore [hunk 2]
```diff
@@ -44,4 +44,6 @@ e2e-test/image/e2e/manifests/
 *.o
 
 __pycache__
-*.pyc
\ No newline at end of file
+*.pyc
+
+docs/
```
Comment (YangKeao) [inline .gitignore:49]: Why should we ignore the `docs`? (and it seems that the `docs` folder doesn't exist)

Comment (cwen0) [reply .gitignore:49]: When I run some build commands, this directory may be generated.

Comment (YangKeao) [reply .gitignore:49]: Which build command?

Comment (cwen0) [reply .gitignore:49]: I am trying to reproduce it.

Comment (cwen0) [reply .gitignore:49]: I can't reproduce this issue. I will close this issue now.

**Pull request closed without merging**